main/src/cgeo/geocaching/connector/gc/GCMap.java

```diff
@@ -148,7 +148,7 @@ public static SearchResult searchByFilter(@NonNull final GeocacheFilter filter,
             fillForBasicFilter(baseFilter, search);
         }
 
-        return GCWebAPI.searchCaches(search);
+        return GCWebAPI.searchCaches(search, true);
```

Comment:
shouldn't including GCVotes depend on the system setting for GCVote connector being activated?

Reply:
Actually it was always like that:
In the GCVote settings the default is to load the ratings unless the user explicitly switches it off.
Autorizing to post votes is independant from that.

However that mentioned setting for loading votes should be respected.

Reply:
Yes, that's what I meant. If someone switches GCVote setting of, no http(s) call to GCVote should be made.

Reply:
The switch in the settings is respected further down the call stack in the GCVote class itself

Reply:
ah, thanks for clarifying 👍